setup.py

```diff
@@ -1,19 +1,23 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+import os
+from setuptools import setup, find_packages
```

Comment:
I switched to setuptools since it provides `find_packages` and is now the standard way to proceed (it should be available everywhere, as many packages already require it by default).

Reply:
Thanks!